build/__variables.sh

```diff
@@ -74,7 +74,7 @@ declare -r ACR_AZURE_FUNCTIONS_JAMSTACK_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:azfun
 declare -r ACR_BUILD_FULL_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:full"
 declare -r ACR_BUILD_LTS_VERSIONS_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:lts-versions"
 declare -r ACR_BUILD_GITHUB_ACTIONS_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:github-actions"
-declare -r ACR_BUILD_VSO_FOCAL_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:vso-focal"
+declare -r ACR_BUILD_VSO_FOCAL_IMAGE_NAME="$ACR_BUILD_IMAGES_REPO:ubuntu-vso-focal"
```

Comment:
Should the format of the tag here be `vso-ubuntu-focal`? From the changes made in the `buildBuildImages.sh` script, it looked like we were doing `<type>-<OS>-<flavor>`, but correct me if I'm wrong.

Reply:
Yeah great point. I'm not sure why but I guess I just assumed the flavor was `vso-focal` even though its `focal-scm` in our code. I'll fix